zip-0340.md

```diff
@@ -0,0 +1,537 @@
+
+The final result of the election is a list of valid ballots that have:
+- a value in ZEC
+- a message
+
+The content of the message depends on the type of referendum.
+The acceptable options should be defined by EB. For example, for a 
+yes/no question, it could simply be a value of 0 or 1.
+
+The EB could decide to accept write-in votes too. The format of the 
+message is not part of this ZIP.
+
+The value of the ballot MUST be an amount of ZEC that the ZH
+can spend at the snapshot height. 
+
+The ZH is not obligated to vote with all the ZEC that they have.
+But it MUST be impossible to use the same coins to vote
+twice on the same election.
+
+It MUST be possible to use the same ZEC on DIFFERENT elections.
+
+**See Privacy section on important caveats regarding concurrent
+voting.**
+
+## Other Uses Cases
+
+### Airdrop
+This ZIP can be used to perform air-drops. An organization
+that wants to give their tokens/coins based on an amount
+of ZEC can organize an election. The ballots contain
+the ZH's address where they want to receive the airdrop.
+
+### Proof of Balance
+In this case, the ZH sends an empty message because the
+ballot value is all that matters.
+
+# Requirements
+
+## Participants / Actors
+
+### Election Board
+
+The Election Board (EB) is the entity in charge of organizing
+the election (E). 
+- The EB MUST have a website secured by https that clearly
+reflects their identity. 
+- They MUST make their announcements 
+through their website. 
+- The EB SHOULD have a forum for election related discussions
+- The EB MUST set the election parameters described in a later
+section
+- The EB MUST publish the election parameters on their website
+- The election parameters are immutable. If the EB
+wants to make a modification, another election MUST be created
+
+### Voter (ZH)
+
+Anyone who has *spending authority* of ZEC 
+(either transparent or shielded) can 
+participate.
+
+- Ballots MUST have a ZEC value
+- Ballots MUST have a message in the format defined by the Election
+- Ballots MUST include a proof of spending authority for the
+amount specified, valid at the snapshot height
+- Ballots MUST not prevent the ZH from spending the coins in a transaction
+- Coins spent before the snapshot height MUST not be eligible for voting
+- Voting MUST not disclose information that would allow someone 
+to use the coins themselves (in a transaction or an election)
+- Ballots MUST follow the format specified in the section Ballot
+- ZEC held in transparent t3 addresses CANNOT be used. They introduce
+most complexity and are not used much
+
+Notes:
+- Having coins on an exchange such as Binance, Bitfinex, etc.
+does not allow you to vote because the coins are under their custody
+- The EB MAY set special rules for the election. For example,
+there MAY be a fee for casting a vote or a minimum ZEC requirement
+
+## Privacy
```

Comment:
This section currently describes how privacy issues could be solved. The ZIP will eventually instead need to address how privacy issues _are_ solved.

Most of this content should be moved to a Security and Privacy Considerations section, that will evolve as the ZIP draft is updated to address privacy issues. The only paragraph that should remain here is the one that includes conformance requirements.

If there are open questions about how to solve concerns that are given in the Security and Privacy Considerations section, add an Open Questions section outlining them (that can become a topic for discussion).